index.html

```diff
@@ -1144,7 +1144,7 @@ <h3 class="t subtitle" data-t="share.options"></h3>
       src="node_modules/jquery-ui/dist/jquery-ui.js"
       type="module"
     ></script>
-    <script src="./js/index.js" type="module"></script>
+    <script src="./js/index.ts" type="module"></script>
```

Comment:
No. You do not serve a ts file to a browser

Reply:
I miss a build step to generate the js file.

Reply:
Vite automatically compiles TypeScript, see https://vitejs.dev/guide/features.html#typescript

#589

Reply:
Thanks.